src/Database/QueryBuilder/MysqlQueryBuilder.php

```diff
@@ -51,6 +52,7 @@ protected function typeMap(): array
 
     protected $defaultLength = [
         Column::TYPE_STRING => 255,
+        Column::TYPE_BIT => 32,
```

Comment:
Is there any reason why to use 32? Isn’t it missing in pgsql query builder too?

Reply:
> Is there any reason why to use 32?
No, this value can be any in the range 1-64 for mysql and with no limit (in any case, I didn't find them in the doc) for pgsql.

> Isn’t it missing in pgsql query builder too?
Oh yes, missed the default length in pgsql query builder.